### DISCUSSION TITLE
Can we add compatability with liveblocks or supabase for support for realtime collaboration?

### DISCUSSION THREAD

**Discussion by leasemagnets** [yoopta-editor/Yoopta-Editor#17] [Ideas]
Can we add compatability with liveblocks or supabase for support for realtime collaboration?

links:
https://liveblocks.io/use-cases/collaborative-creative-tools
[supabase.com
](https://supabase.com/docs/guides/realtime/broadcast)

I think the only real thing that yopta editor seems to be missing is support for real time collaboration.

**Comment by Darginec05** [comment]
Not sure if it will be available in v2, but of course I have plans for this :)

**Comment by maZahaca** [comment]
Hey @leasemagnets! We're under development collaborative mode for the editor based on this [issue](https://github.com/Darginec05/Yoopta-Editor/issues/127)

**Comment by sreekanthedayar** [comment]
Why to go with liveblocks,  hocuspocus along with Yjs will be more appealing as its fully opensource.

**Comment by maxim-semenov-hrworks** [comment]
Any progress here? I'm also interested in hocuspocus integration 👍

**Comment by Darginec05** [comment]
@leasemagnets @maZahaca @sreekanthedayar @oranginaround hey comrades, I finally shipped collaboration mode via package @yoopta/collaboration

Try it → [yoopta.dev/playground/collaboration](https://yoopta.dev/playground/collaboration)
Docs → [docs.yoopta.dev/core/collaboration](https://docs.yoopta.dev/core/collaboration)
